### PR TITLE
Implement Disappearing Messages

### PR DESCRIPTION
**Pull request by Jikstra**
Screenshot_20200712_144416
Screenshot_20200712_144450

### CODE REVIEW

**Comment by pabzm** [comment]
Please don't merge this into master at this moment.

We're trying to focus on stability and bug fixes for the release of v1.0. Adding code for features that haven't even been merged into the core (and are not in focus there, either) isn't helping with that.

**Comment by Simon-Laux** [comment]
Why are there translation updates in this pr?

**Comment by Jikstra** [comment]
@Simon-Laux because i needed translations for this pr?

**Comment by Simon-Laux** [comment]
This makes it harder to review, I would have preferred a separate pr, that gets merged and this one would have gotten re-based then.

File: src/renderer/components/dialogs/index.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import React from 'react'
```
Comment (Simon-Laux) [inline src/renderer/components/dialogs/index.tsx:1]: what is this new file about? is it a leftover from a rebase?

File: src/renderer/components/Menu.tsx [hunk 2]
```diff
@@ -128,10 +136,21 @@ export default function DeltaMenu(props: { selectedChat: FullChat }) {
           text={tx('menu_unmute')}
         />
       ),
-      <Menu.Divider key='divider2' />,
+      <SettingsContext.Consumer>
+        {({ desktopSettings }) =>
+          desktopSettings.enableDisappearingMessages && (
+            <DeltaMenuItem
+              key='disappearing'
+              text={tx('ephemeral_messages')} // TODO: Translate
```
Comment (Simon-Laux) [inline src/renderer/components/Menu.tsx:144]:
```suggestion
              text={tx('ephemeral_messages')}
```

**Pull request merged**